gateway/src/main/java/org/georchestra/gateway/filter/headers/HeaderFiltersConfiguration.java

```diff
@@ -78,15 +84,17 @@ public class HeaderFiltersConfiguration {
      * General purpose {@link GatewayFilterFactory} to remove incoming HTTP request
      * headers based on a Java regular expression
      */
-    public @Bean RemoveHeadersGatewayFilterFactory removeHeadersGatewayFilterFactory() {
-        return new RemoveHeadersGatewayFilterFactory();
+    public @Bean RemoveHeadersGatewayFilterFactory removeHeadersGatewayFilterFactory(
+            GatewayConfigProperties configProps) {
+        return new RemoveHeadersGatewayFilterFactory(configProps);
```

Comment:
I am still confused about this change, and cannot get why we could have the bean `@Autowired`